spec/hscode/config/bootstrap_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+describe Bootstrap::Config do
+  describe '#get' do
+    let(:root) { File.expand_path('../../../../', __FILE__) }
+
+    it 'gets all environment variables' do
+      File.open("#{root}/config/env.yaml", 'w+') do |f|
```

Comment:
Could we wrap this in a before hook and the have a corresponding after hook to clean up the write? Seems like a long shot though? If we have actual configs, will they be overwritten?